send.php

```diff
@@ -29,64 +29,73 @@
 
 $dir = dirname($file);
 $old = $file;
-$file = $dir . '/analytics-' . rand() . '.log';
 
-if(!file_exists($old)) {
-  print("file: $old does not exist");
-  exit(0);
-}
+$newFileName = 'analytics-' . rand() . '.log';
+$newFile = $dir . '/' . $newFileName;
 
-if (!rename($old, $file)) {
-  print("error renaming from $old to $file\n");
-  exit(1);
+if(file_exists($old)) {
```

Comment:
So if old file does not exist, we won't print anything? Could be nice to still do it for debugging.

Reply:
Don't think it's a "broken" scenario in any way: there's a very legitimate scenario that the new file simply has not been created yet. Generally the printing seems to be output status details only.
Could print it, but I'm not sure if it adds anything necessarily.